multi-int-store/src/test/java/com/connexta/multiintstore/MultiIntStoreIntegrationTest.java

```diff
@@ -32,7 +34,8 @@
 public class MultiIntStoreIntegrationTest extends MultiIntStoreIntegrationTestContainers {
 
   @Autowired private WebApplicationContext wac;
-  @Autowired private CommonSearchTermsRepository cstRepository;
+  @Autowired private IndexedMetadataRepository searchRepository;
+  @Autowired private SearchService searchService;
```

Comment:
✏️ I know this is a test class, but we can use an `Autowired` constructor for consistency
```java
private final WebApplicationContext wac;
private final IndexedMetadataRepository searchRepository;
private final SearchService searchService;

@Autowired
public MultiIntStoreIntegrationTest(WebApplicationContext wac, IndexedMetadataRepository searchRepository, SearchService searchService) {
  this.wac = wac;
  this.searchRepository = searchRepository;
  this.searchService = searchService;
}
```